src/App.js

```diff
@@ -1,27 +1,27 @@
-import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import React, { useState, useEffect } from "react";
+import { getAllData } from "./util/index";
+import Home from "./component/Home/Home";
 
-const URL = 'http://localhost:8000/api/v1/';
+const URL = "http://localhost:8000/api/v1/";
 
 function App() {
-
-const [message, setMessage] = useState(''); 
+  const [message, setMessage] = useState("");
 
   useEffect(() => {
-
     (async () => {
```

Comment:
I get an error in the console from this async function, if this is a placeholder might be best to comment out lines 11-14 to avoid the errors in console.

Reply:
I have no issues like that. This Async function wasn’t created by me. And Anna removed this in her PR. Can you double check my code?

Reply:
Right so the error is in the console. Error occurs in App.js, ok as long as you are all aware where this could lead to merge conflict later on, just communicate which lines of code cause these errors and which version of the file is up to date.

<img width="1215" alt="Screenshot 2023-05-17 at 11 00 33 PM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/39227111/628af822-5b13-421c-bd79-f123d7b08130">